src/pages/patientView/clinicalInformation/lib/clinicalAttributesUtil.js

```diff
@@ -207,21 +207,132 @@ function getSpanElementsFromCleanData(
     clinicalAttributesCleanDerived,
     cancerStudyId
 ) {
-    let spans = [];
-    return Object.keys(clinicalAttributesCleanDerived).map(key => {
-        let value = clinicalAttributesCleanDerived[key];
-        return (
-            <span
-                is
-                class="clinical-attribute"
-                attr-id={key}
-                attr-value={value}
-                study={cancerStudyId}
-            >
-                {value}
-            </span>
-        );
-    });
+    // Order patient clinical attributes
+    const getOrder = key => {
+        const map = {
```

Comment:
I'm not a huge fan of having these big `const`s just hanging around in these functions. I think it makes things a little less readable. Maybe they'd be better off at the top of this file, with better names. Don't change it yet, though, I'm curious what @alisman thinks.

Reply:
Thinking about this a little more, my preferred solution would be to pull these consts into their own file. clinicalAttributesStyleConsts.js or something.

Reply:
I moved the const functions into their own file and documented them with comments just now